File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int, int> mp;
        for(auto n: nums) mp[n]++;
        vector<pair<int, int>> fp;
        for(auto &[k, v] : mp) fp.push_back({v, k});
        priority_queue<pair<int, int>> pq(fp.begin(), fp.end());
        vector<int> ans;
        for(int i = 0; i < k; i++){
            ans.push_back(pq.top().second);
            pq.pop();
        }
        return ans;
    }
};

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        l=0
        ans = []
        for r in range(k-1, len(nums)):
            ans.append(max(nums[l : r + 1]))
            l += 1
        return ans